### PR TITLE
PR than i made

### PR DESCRIPTION
**Pull request by louis1706**
Thanks Ika for the Help on CustomAmmo/CategoryLimit
Make than every FpcRole trigger EscapeEvent
Added `Player::FallDamage` property
Added `Player::AmmoLimit`&`Player::CategoryLimit`
Added `Recontainer::EngagedGeneratorCount`

### CODE REVIEW

File: EXILED/Exiled.API/Features/Roles/FpcRole.cs [hunk 1]
```diff
@@ -55,6 +58,19 @@ public RelativePosition RelativePosition
...
 
+        /// <summary>
+        /// Gets or sets a value indicating whether if the player should get <see cref="Enums.DamageType.Falldown"/> damage.
+        /// </summary>
+        public bool IsFallDamageEnable
+        {
+            get => FirstPersonController.FpcModule.Motor._enableFallDamage;
+            set
+            {
+                enableFallDamageField ??= AccessTools.Field(typeof(FpcMotor), nameof(FpcMotor._enableFallDamage));
```
Comment (VALERA771) [inline EXILED/Exiled.API/Features/Roles/FpcRole.cs:69]: No publicized?

Comment (louis1706) [reply EXILED/Exiled.API/Features/Roles/FpcRole.cs:69]: It's get only

Comment (louis1706) [reply EXILED/Exiled.API/Features/Roles/FpcRole.cs:69]: was readonly `private readonly bool _enableFallDamage;` it's not publicized

**Review by Misfiy — APPROVED** [review]

**Review by Jesus-QC — CHANGES_REQUESTED** [review]
Files being deleted

**Review by Misfiy — APPROVED** [review]

**Review by Vladislav-CS — APPROVED** [review]

**Pull request merged**